chart/flux/templates/_helpers.tpl

```diff
@@ -41,3 +41,22 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create a custom repositories.yaml for Helm
+*/}}
+{{- define "flux.customRepositories" -}}
+apiVersion: v1
+generated: 0001-01-01T00:00:00Z
+repositories:
+{{- range .Values.helmOperator.configureRepositories.repositories }}
+- name: {{ .name }}
+  url: {{ .url }}
+  cache: /var/fluxd/helm/repository/cache/{{ .name }}-index.yaml
+  caFile: ""
+  certFile: ""
+  keyFile: ""
+  password: {{ .password }}
```

Comment:
Does this work with empty string? Shouldn't be `"{{ .password }}"` and `"{{ .username }}"` ?

Reply:
You're totally right. I also made the URL and name required.